src/microbe_stage/editor/MicrobeEditor.cs

```diff
@@ -248,6 +248,21 @@ public float Rigidity
         }
     }
 
+    [JsonProperty]
+    public float Aggression { get; private set; }
+
+    [JsonProperty]
+    public float Opportunism { get; private set; }
+
+    [JsonProperty]
+    public float Fear { get; private set; }
+
+    [JsonProperty]
+    public float Activity { get; private set; }
+
+    [JsonProperty]
+    public float Focus { get; private set; }
+
```

Comment:
Since you've already added an enum for this, you could cut down on lines by converting these to a dictionary using the enum values as keys. This would also ensure that any code using the personality variables will always include all of them, even if we end up adding or removing one in the future.

Reply:
Is that really a good idea? Other code also has these properties split out and this makes reading the properties way easier.

Reply:
It might slightly be easier to handle loading an old save with a single object that holds all of these, that way on editor load the object can be created if it is null (loaded from an old save or a freshly entered editor) or if it is present used as is.